airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -147,7 +147,7 @@ protected void acceptTracked(final AirbyteMessage message) throws Exception {
       // are serialized again when writing to
       // the destination
       long messageSizeInBytes = ByteUtils.getSizeInBytesForUTF8CharSet(Jsons.serialize(recordMessage.getData()));
-      if (bufferSizeInBytes + messageSizeInBytes >= maxQueueSizeInBytes) {
+      if (bufferSizeInBytes + messageSizeInBytes > maxQueueSizeInBytes) {
```

Comment:
why is this changed too? should this also be `=`?

Reply:
As for me it's real bug in code itself which I discovered fixing tests.
We need to FLUSH buffer only if 
`bufferSizeInBytes +  messageSizeInBytes > maxQueueSizeInBytes`

because if

`bufferSizeInBytes +  messageSizeInBytes = maxQueueSizeInBytes`

we still has to keep buffer not flushed

it's not my logic
it's what I see according to tests